### PR TITLE
Explicit rdb vars

### PR DESCRIPTION
**Pull request by ashtul**
completes PR #503 
Saving the chunk struct vars individually

### CODE REVIEW

**Review by rafie — APPROVED** [review]

File: src/module.c [hunk 1]
```diff
@@ -597,7 +597,7 @@ int ReplySeriesRange(RedisModuleCtx *ctx,
     }
 
     RedisModule_ReplyWithArray(ctx, REDISMODULE_POSTPONED_ARRAY_LEN);
-    if (aggObject == TS_AGG_NONE) {
+    if (aggObject == NULL) {
```
Comment (danni-m) [inline src/module.c:600]: @ashtul @rafie how is this related to the PR?

Comment (ashtul) [reply src/module.c:600]: It isn't.
It's a small mistake since `aggObject` is a pointer but `TS_AGG_NONE` value is `0` so we didn't get a warning.
